core/config/default-config.js

```diff
@@ -123,54 +123,49 @@ const UIStrings = {
 
 const str_ = i18n.createIcuMessageFn(import.meta.url, UIStrings);
 
-// Ensure all artifact IDs match the typedefs.
-/** @type {Record<keyof LH.FRArtifacts, string>} */
+/** @type {SelfMap<keyof LH.FRArtifacts>} */
 const artifacts = {
-  DevtoolsLog: '',
-  Trace: '',
-  Accessibility: '',
-  AnchorElements: '',
-  CacheContents: '',
-  ConsoleMessages: '',
-  CSSUsage: '',
-  Doctype: '',
-  DOMStats: '',
-  EmbeddedContent: '',
-  FontSize: '',
-  Inputs: '',
-  FullPageScreenshot: '',
-  GlobalListeners: '',
-  IFrameElements: '',
-  ImageElements: '',
-  InstallabilityErrors: '',
-  InspectorIssues: '',
-  JsUsage: '',
-  LinkElements: '',
-  MainDocumentContent: '',
-  MetaElements: '',
-  NetworkUserAgent: '',
-  OptimizedImages: '',
-  PasswordInputsWithPreventedPaste: '',
-  ResponseCompression: '',
-  RobotsTxt: '',
-  ServiceWorker: '',
-  ScriptElements: '',
-  Scripts: '',
-  SourceMaps: '',
-  Stacks: '',
-  TagsBlockingFirstPaint: '',
-  TapTargets: '',
-  TraceElements: '',
-  ViewportDimensions: '',
-  WebAppManifest: '',
-  devtoolsLogs: '',
-  traces: '',
+  DevtoolsLog: 'DevtoolsLog',
+  Trace: 'Trace',
+  Accessibility: 'Accessibility',
+  AnchorElements: 'AnchorElements',
+  CacheContents: 'CacheContents',
+  ConsoleMessages: 'ConsoleMessages',
+  CSSUsage: 'CSSUsage',
+  Doctype: 'Doctype',
+  DOMStats: 'DOMStats',
+  EmbeddedContent: 'EmbeddedContent',
+  FontSize: 'FontSize',
+  Inputs: 'Inputs',
+  FullPageScreenshot: 'FullPageScreenshot',
+  GlobalListeners: 'GlobalListeners',
+  IFrameElements: 'IFrameElements',
+  ImageElements: 'ImageElements',
+  InstallabilityErrors: 'InstallabilityErrors',
+  InspectorIssues: 'InspectorIssues',
+  JsUsage: 'JsUsage',
+  LinkElements: 'LinkElements',
+  MainDocumentContent: 'MainDocumentContent',
+  MetaElements: 'MetaElements',
+  NetworkUserAgent: 'NetworkUserAgent',
+  OptimizedImages: 'OptimizedImages',
+  PasswordInputsWithPreventedPaste: 'PasswordInputsWithPreventedPaste',
+  ResponseCompression: 'ResponseCompression',
+  RobotsTxt: 'RobotsTxt',
+  ServiceWorker: 'ServiceWorker',
+  ScriptElements: 'ScriptElements',
+  Scripts: 'Scripts',
+  SourceMaps: 'SourceMaps',
+  Stacks: 'Stacks',
+  TagsBlockingFirstPaint: 'TagsBlockingFirstPaint',
+  TapTargets: 'TapTargets',
+  TraceElements: 'TraceElements',
+  ViewportDimensions: 'ViewportDimensions',
+  WebAppManifest: 'WebAppManifest',
+  devtoolsLogs: 'devtoolsLogs',
+  traces: 'traces',
 };
```

Comment:
Drive-by: since this config serves as our prime example, I thought this would help readability